Pipfile

```diff
@@ -11,11 +11,11 @@ python_version = "3"
 constant-sorrow = ">=0.1.0a9"
 bytestring-splitter = ">=2.4.0"
 hendrix = ">=4.0"
-nucypher-core = ">=0.10.0"
+nucypher-core = {git="https://github.com/nucypher/nucypher-core.git", ref="7f3a7f21f2f9bdc0395e256025ec9af0d53723aa", subdirectory="nucypher-core-python"}
```

Comment:
Just a reminder to update this before merging, once the new version of `nucypher-core` is released.